### PR TITLE
Starting laws retouch

### PR DESCRIPTION
**Pull request by Mercur-3**
Please Please Please

### CODE REVIEW

**Review by JackEPepper — CHANGES_REQUESTED** [review]
Need error fixing

File: common/history/countries/den - denmark.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		# Laws
 		effect_create_law_package = {
 			headOfGovernment = hereditary
-			distributionOfPower = representative_democracy
+			distributionOfPower = consensus_democracy
```
Comment (DZappala) [inline common/history/countries/den - denmark.txt:11]: Most nations don't start with the tech for consensus democracy u less this requirement was changed

File: common/history/countries/abu - trucial states.txt
Comment (DZappala) [inline common/history/countries/abu - trucial states.txt:1]: todo: needs refactoring

File: common/history/countries/ade - aden protectorate.txt
Comment (DZappala) [inline common/history/countries/ade - aden protectorate.txt:1]: todo: needs refactoring

File: common/history/countries/afh - afganistan.txt
Comment (DZappala) [inline common/history/countries/afh - afganistan.txt:1]: todo: needs refactoring

File: common/history/countries/alb - albania.txt
Comment (DZappala) [inline common/history/countries/alb - albania.txt:1]: todo: needs refactoring

File: common/history/countries/arg - argentina.txt
Comment (DZappala) [inline common/history/countries/arg - argentina.txt:1]: todo: needs refactoring

File: common/history/countries/ast - australia.txt
Comment (DZappala) [inline common/history/countries/ast - australia.txt:1]: removing a law from the effect will cause the entire effect to not work. This change needs to be undone.

File: common/history/countries/aus - austria.txt [hunk 2]
```diff
@@ -3,6 +3,36 @@
 		# Politics
 		#todo IGs and Ideologies
 		# Tech and Tax
+		set_tax_level = high
+		effect_create_law_package = {
+			headOfGovernment = council_elective
```
Comment (DZappala) [inline common/history/countries/aus - austria.txt:8]: soviet occupied austria should be 
administrative as it's a military occupation.

File: common/history/countries/azr - azerbaijan.txt [hunk 3]
```diff
@@ -21,17 +21,17 @@
...
 			childrensRights = compulsory_education
 			rightsOfWomen = legal_guardianship
 			familyPlanning = criminalized_birth_control
 			welfare = welfare_state
-			migration = open_borders
+			migration = selective_immigration
 			biotech = animal_testing_only
 			recreationalDrugs = tobacco_allowed
 		}
+		activate_law = law_type:law_no_wmds
```
Comment (DZappala) [inline common/history/countries/azr - azerbaijan.txt:34]: activate law can't be moved out of effect.

File: common/history/countries/bel - belgium.txt [hunk 4]
```diff
@@ -29,9 +29,9 @@
 			familyPlanning = no_family_planning
 			welfare = income_supplementation
 			migration = selective_immigration
-			biotech = theoretical_research_only
 			recreationalDrugs = alcohol_allowed
 		}
+		activate_law = law_type:law_regulated_human_trials
```
Comment (DZappala) [inline common/history/countries/bel - belgium.txt:34]: activate law can't be out of effect.

File: common/history/countries/bhn - bahrain.txt [hunk 5]
```diff
@@ -30,9 +30,9 @@
 			familyPlanning = no_family_planning
 			welfare = no_welfare
 			migration = selective_immigration
-			biotech = theoretical_research_only
 			recreationalDrugs = tobacco_allowed
 		}
+		activate_law = law_type:law_regulated_human_trials
```
Comment (DZappala) [inline common/history/countries/bhn - bahrain.txt:35]: activate law can't be out of effect.

File: common/history/countries/bhu - bhutan.txt
Comment (DZappala) [inline common/history/countries/bhu - bhutan.txt:1]: Missing starting laws. all laws must be set. 
todo: needs refactoring

File: common/history/countries/bol - bolivia.txt [hunk 6]
```diff
@@ -30,9 +27,12 @@
 			familyPlanning = no_family_planning
 			welfare = no_welfare
 			migration = selective_immigration
-			biotech = theoretical_research_only
 			recreationalDrugs = tobacco_allowed
 		}
+		activate_law = law_type:law_penal_correction
+		activate_law = law_type:law_executive
+		activate_law = law_type:law_professional_police
+		activate_law = law_type:law_regulated_human_trials
```
Comment (DZappala) [inline common/history/countries/bol - bolivia.txt:35]: activate law can't be added outside of effect.

File: common/history/countries/boz - british occupation zone.txt [hunk 7]
```diff
@@ -7,15 +7,16 @@
 		effect_currency_dem = yes
 		#laws
 
+		activate_law = law_type:law_penal_correction
+		activate_law = law_type:law_professional_police
```
Comment (DZappala) [inline common/history/countries/boz - british occupation zone.txt:11]: activate law can't be outside effect

File: common/history/countries/brd - Burundi.txt [hunk 8]
```diff
@@ -6,15 +6,17 @@
 		# capitals\
 		# TODO
 		#laws
+
+		activate_law = law_type:law_penal_correction
+		activate_law = law_type:law_professional_police
```
Comment (DZappala) [inline common/history/countries/brd - Burundi.txt:11]: activate law can't be outside effect

File: common/history/countries/ddr - german democratic republic.txt [hunk 9]
```diff
@@ -7,22 +7,22 @@
 
 		effect_create_law_package = {
 			headOfGovernment = administrative
-			distributionOfPower = representative_democracy
+			distributionOfPower = anocracy
```
Comment (DZappala) [inline common/history/countries/ddr - german democratic republic.txt:10]: mil admins should be either autocracy or oligarchy.

File: common/history/countries/leb - lebanon.txt [hunk 10]
```diff
@@ -5,7 +5,7 @@
 		# Tech and Tax
 		effect_starting_technology_tier_4_tech = yes
 		effect_create_law_package = {
-			headOfGovernment = administrative
+			headOfGovernment = executive
```
Comment (DZappala) [inline common/history/countries/leb - lebanon.txt:8]: Was this not a military occupation by the British at this time?

File: common/history/countries/nrk - north korea.txt [hunk 11]
```diff
@@ -7,7 +7,7 @@
 		effect_corruption_medium = yes
 		effect_create_law_package = {
 			headOfGovernment = council_elective
-			distributionOfPower = oligarchy
+			distributionOfPower = anocracy
```
Comment (DZappala) [inline common/history/countries/nrk - north korea.txt:10]: if this is occupied north Korea they should be administrative autocracy.

File: common/history/countries/stk - south korea.txt [hunk 12]
```diff
@@ -7,7 +7,7 @@
 		effect_corruption_medium = yes
 		effect_create_law_package = {
 			headOfGovernment = administrative
-			distributionOfPower = representative_democracy
+			distributionOfPower = anocracy
```
Comment (DZappala) [inline common/history/countries/stk - south korea.txt:10]: mil admins should be autocracy or oligarchy.

**Review by DZappala — CHANGES_REQUESTED** [review]

**Review by DZappala — APPROVED** [review]

**Review by JackEPepper — APPROVED** [review]

**Pull request merged**